examples/publish_module/sources/publish_module.move

```diff
@@ -0,0 +1,17 @@
+        // Fixme: this module is compiled with my local account, so it may failed in other environment 
+        //  or with other signers.
+        let module_bytes: vector<u8> = x"a11ceb0b060000000b010006020608030e26043406053a32076c8c0108f8014006b802220ada02050cdf02560db5030200000101010200030c00020400000005000100000600010000070201000008030400010907080108010a09010108010b0a0b0108040605060606010708010002070801060c0106080101030107080001080002070801050107090003070801060c090002060801050106090007636f756e7465720f6163636f756e745f73746f726167650f73746f726167655f636f6e7465787407436f756e7465720e53746f72616765436f6e7465787408696e63726561736509696e6372656173655f04696e69740576616c756511676c6f62616c5f626f72726f775f6d75740e676c6f62616c5f6d6f76655f746f0d676c6f62616c5f626f72726f77ce0c46d8ef427442d6c3b05bcc4ff6d33290f8d3ef8abdf96e344466b7303a9d00000000000000000000000000000000000000000000000000000000000000020520ce0c46d8ef427442d6c3b05bcc4ff6d33290f8d3ef8abdf96e344466b7303a9d00020108030001040001030b0011010201010000050d0b00070038000c010a01100014060100000000000000160b010f0015020200000001060b000b0106000000000000000012003801020301000001060b000700380210001402000000";
```

Comment:
how about pass the module_bytes and entry function argument?

Reply:
Entry function does not support  nested vector . But this may be a workround for a single file module.

Reply:
> Entry function does not support nested vector . But this may be a workround for a single file module.

Entry function does support nested vector, but CLI does not support.